forest/sycamore/responses.py

```diff
@@ -103,6 +108,7 @@ def agg_changed_answers_summary(
             Summary of how an individual answered each question, with their
             most common answer, time to answer, etc
     """
+    agg = agg.copy()
```

Comment:
Dataframe is copied into itself?

Reply:
I did this because dataframes are mutable, but I'm realizing that maybe that didn't solve the problem.